packages/runtime/test/src/lib/txn.ts

```diff
@@ -170,10 +170,10 @@ describe("Convert encoded Txn to ExecParams", function () {
 				type: types.TransactionType.DeployApp,
 				approvalProgram: "counter-approval.teal",
 				clearProgram: "clear.teal",
-				globalBytes: 10,
-				globalInts: 10,
-				localBytes: 10,
-				localInts: 10,
+				globalBytes: 1,
+				globalInts: 1,
+				localBytes: 1,
+				localInts: 1,
```

Comment:
why changing all of this?

Reply:
i think this would throw error otherwise now. local state entries total to 20 (max 16 allowed)